pgx/src/pgx_value.ml

```diff
@@ -1,9 +1,15 @@
 open Sexplib0.Sexp_conv
 open Pgx_aux
 
-type t = string option [@@deriving sexp_of]
+let compare_float = Float.compare
+let compare_int = Int.compare
+let compare_option = Stdlib.Option.compare
```

Comment:
since we are using base, we can just `open Base` and use`Option.compare`. We also won't need to open sexplib0 if we are opening base

Reply:
I think we should do that at some point, but it will require more changes because Base shadows and deprecates a lot of things.